tests/test_unhandled_revisions.py

```
@@ -0,0 +1,558 @@
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore", UnhandledRevisionWarning)
+            with _open(path) as doc:
+                doc.accept_all()
+                doc.save(out)
+            with _open(out) as reopened:
+                assert reopened.list_revisions() == []  # nothing left to adjudicate
+                rows = reopened.list_unhandled_revisions()
+        assert [r.tag for r in rows] == ["w:pPrChange"]
+
+    @pytest.mark.parametrize(
+        "method,expected_types",
```

Ruff (0.16.2)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)